src/main/java/internet/market/security/AuthenticationServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package internet.market.security;
+
+import internet.market.exceptions.AuthenticationException;
+import internet.market.lib.Inject;
+import internet.market.lib.Service;
+import internet.market.model.User;
+import internet.market.service.UserService;
+
+@Service
+public class AuthenticationServiceImpl implements AuthenticationService {
+    @Inject
+    UserService userService;
+
+    @Override
+    public User login(String login, String password) throws AuthenticationException {
+        User user = userService.findByLogin(login)
+                .orElseThrow(() ->
+                        new AuthenticationException("Register first or enter correct login"));
```

Comment:
```suggestion
                        new AuthenticationException("Wrong password or login"));
```
let's not give any clue to malefactors)

Reply:
done